NEWS.md

```diff
@@ -1,3 +1,10 @@
+# uptasticsearch development version
+
+## Features
+
+### Full support for ES7.x
```

Comment:
Maybe just "Support" without "Full"? Since new features aren't accounted for yet?

Reply:
yeah good call

Reply:
still have to do this I think

Reply:
ugh you're right. I want Bitbucket tasks back :(